software/duet_config_files/duet2/rrf2/macros/config_checks.g

```diff
@@ -0,0 +1,73 @@
+; Machine Prechecks
+M84
+M291 R"Machine Prechecks" P"Please move printhead to center of bed, and bed to middle height, and tool changer axis to middle." S3
```

Comment:
I would make a note that they can do this now without turning off the machine. Something like:

"Motors have been disabled and can now be rotated by hand. <Rest of original message>"

Let's also refer to the printhead as the carriage if it doesn't have tools.